File: 1716-calculate-money-in-leetcode-bank/1716-calculate-money-in-leetcode-bank.c
int totalMoney(int n){
    int money = 0,i = 0,add = 1;
    
    while(i < n){
        if(i % 7 == 0)
            add = (i / 7) + 1;
        money += add;
        add++;
        i++;
    }
    return money;
}

File: 1716-calculate-money-in-leetcode-bank/1716-calculate-money-in-leetcode-bank.swift
class Solution {
    func totalMoney(_ n: Int) -> Int {
        var money = 0
        var i = 0
        var add = 1
    
        while i < n {
            if i % 7 == 0 {
                add = (i / 7) + 1
            }
            money += add
            add += 1
            i += 1
        }

        return money
    }
}